clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -31,6 +31,20 @@ struct CIROpAsmDialectInterface : public OpAsmDialectInterface {
   using OpAsmDialectInterface::OpAsmDialectInterface;
 
   AliasResult getAlias(Type type, raw_ostream &os) const final {
+    if (auto intType = dyn_cast<cir::IntType>(type)) {
+      // We only provide alias for standard integer types (i.e. integer types
+      // whose width is a power of 2 and at least 8).
```

Comment:
The incubator is only checking `width % 8 == 0`, but that creates aliases for things like `!cir.int<s, 48>`, which I don't think is what was intended.

Reply:
Makes total sense.